docs/help/getting-started.md

```diff
@@ -7,14 +7,14 @@ Bump.sh connects people with APIs. From your API specifications, Bump.sh generat
 ### Step 1: Create your space/documentation
 
 After creating your account, you'll be asked [to create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation.
-By default, your documentation URL will use the same name: you can customize it in the slug field. You can also use [custom domains](help/custom-domains.md), which we'll explain later.
+By default, your documentation URL will use the same name: you can customize it in the slug field. You can also use [custom domains](custom-domains.md), which we'll explain later.
 
 ![](/files/doc-creation.png)
 
 ### Step 2: Upload your file
 
 If you already have a specification file, you can immediately upload it.
-You can also use our examples to see Bump.sh in action. Finally, our [CLI](help/bump-cli) can also be directly used to jump in.
+You can also use our examples to see Bump.sh in action. Finally, our [CLI](bump-cli) can also be directly used to jump in.
```

Comment:
```suggestion
You can also use our examples to see Bump.sh in action. Finally, our [CLI](bump-cli.md) can also be directly used to jump in.
```

Maybe docusaurus doesn't find the page because of the missing extension?

Reply:
Yes, it was the exact reason. Thanks for the proposal :)